modules/ezcontent_node/modules/ezcontent_landing_page/config/install/core.entity_view_display.node.landing_page.default.yml

```diff
@@ -77,6 +77,22 @@ third_party_settings:
         'User fields': {  }
         core: {  }
       allowed_layouts: {  }
+      whitelisted_blocks:
+        'Chaos Tools': {  }
+        'Content fields':
+          - 'field_block:node:landing_page:uid'
+          - 'field_block:node:landing_page:created'
+          - 'field_block:node:landing_page:body'
+          - 'field_block:node:landing_page:title'
+        Help: {  }
+        Menus: {  }
+        'Moderation Dashboard': {  }
+        'Responsive Preview': {  }
+        System: {  }
+        User: {  }
+        core: {  }
+      blacklisted_blocks: {  }
+    allowed_block_categories: {  }
```

Comment:
Is this config related to Autosave ?

Reply:
No, i have also pushed EZ-195 in the same branch. I have updated the message for more clarity